.gitignore

```diff
@@ -8,3 +8,5 @@ hack/config-provider-*
 _out/
 
 _kubevirtci
+
+.idea/
```

Comment:
This is not related to this project. Please use a [global gitignore](https://stackoverflow.com/questions/7335420/global-git-ignore) to ignore files specific to your IDE